doc/classes/ProjectSettings.xml

```diff
@@ -713,6 +713,8 @@
 		</member>
 		<member name="display/window/stretch/scale" type="float" setter="" getter="" default="1.0">
 		</member>
```

Comment:
```suggestion
		<member name="display/window/stretch/scale" type="float" setter="" getter="" default="1.0">
			The scale factor multiplier to use for 2D elements. This multiplies the final scale factor determined by [member display/window/stretch/mode]. If using the [b]Disabled[/b] stretch mode, this scale factor is applied as-is. This can be adjusted to make the UI easier to read on certain displays.
		</member>
```